packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -634,7 +635,7 @@ describe('TransactionController', () => {
       const controller = newController({ config: { interval: 1337 } });
 
       await wait(100);
-      controller.poll(1338);
+      logRejection(controller.poll(1338));
```

Comment:
In this case we should probably be `await`-ing this promise.

If we want to run the assertion before resolving the promise, we could capture the promise in a variable and `await` it after the assertion. But for the sake of cleanup we probably don't want to leave this running after the test ends.